src/ex3_js-objects-part1/task-01.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+let array = {};
```

Comment:
серьезно?

Reply:
```suggestion
let object= {};
```
:see_no_evil: стыдоба